app/src/main/java/edu/agh/susgame/front/ui/component/menu/MainMenuView.kt

```diff
@@ -36,6 +36,7 @@ private fun MainMenuButton(text: String, onClick: () -> Unit) {
 fun MainMenuView(
     navController: NavController,
 ) {
+//    navController.navigate("${MenuRoute.Game.route}/0") // JUST FOR TESTING
```

Comment:
Do wyrzucenia?

Reply:
dodałem po to, żeby nie musieć po każdym uruchomieniu nawigować do mapy przez wszystkie przyciski, bardzo usprawnia prace, zmienie komentarz zeby bylo czytelniej

Reply:
Bardzo mądre, faktycznie bez sensu jest się przez to przeklikiwać przy każdej zmianie